### DISCUSSION TITLE
IP address of the lima VM

### DISCUSSION THREAD

**Discussion by mirza-ali-ctct** [lima-vm/lima#777] [Q&A] [answered]
Coming from `minikube ip`, the K8 services are accessible on host machine. Is there a similar feature in lima so that we can call services in k8 using lima ip?

**Comment by jandubois** [comment, marked as answer]
If you have configured a LoadBalancer service, then it should be accessible via `localhost`. You would need to configure a shared/bridged interface via `vde_vmnet` to have a separate IP address (and be able to get the external service address from `kubectl get svc`.

**Comment by mirza-ali-ctct** [comment]
Thanks for the awesome reply. I am using k8s.yaml example from `examples` folder and I added following in the yaml 
```yaml
networks:
  - lima: bridged
  ```
but now getting very different issue
```bash
➜  lima limactl start ./k8s.yaml
? Creating an instance "k8s" Proceed with the current configuration
FATA[0001] can't read "/private/etc/sudoers.d/lima": open /private/etc/sudoers.d/lima: no such file or directory
```

**Reply by jandubois** [reply]
You have to create it yourself. Something like

```
limactl sudoers | sudo tee /etc/sudoers.d/lima
```
